### ISSUE TITLE
A project named "_" already exists

### ISSUE THREAD

**Issue by jaredbuttars** [directus/docs#171]
A fresh local install of Directus doesn't get past the installation process with the error:

A project named "_" already exists.

From the error log:

`api[].ERROR: Directus\Exception\ProjectAlreadyExistException: A project named "_" already exists. in <sitepath>\src\core\Directus\Services\ProjectService.php:76`

**Comment by rijkvanzanten** [comment]
How did you install? If the config already exists it's most likely not a fresh install, or you already configured the API separately in another way  👀

**Comment by jaredbuttars** [comment]
Following the instructions at
https://docs.directus.io/getting-started/installation.html#setup

This is the first time Directus of any type has been installed on this particular machine. But even if it had previously been installed the Project Key field is disabled and set to _ as default. I'm not sure why the field is there if it can't be changed.

I didn't have this problem installing on a different machine.

**Comment by jaredbuttars** [comment]
I took additional steps.

- Dropped and then recreated a fresh database
- completely removed all Directus files. I put them back with a fresh git clone.
- set doc root to /public and navigated to /admin
- configured the installation and while install was running I got the message: "Syntax error or access violation: 1071 Specified key was too long"

from the log:

api[].ERROR: PDOException: SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes in <project path>\vendor\robmorgan\phinx\src\Phinx\Db\Adapter\PdoAdapter.php:134
